openslides/core/models.py

```diff
@@ -243,3 +273,61 @@ class Meta:
     def get_session_store_class(cls):
         from .session_backend import SessionStore
         return SessionStore
+
+
+class Message(RESTModelMixin, models.Model):
```

Comment:
Ah this looks like saving redundant data. At least the uuid exists twice: In the projector config field and in this database field. Why is this necessary? You can store the same message even in multiple projectors. If they should live there.

Or you also change the Message projector element and transform this to a normal slide like all others.